src/app/resources/resources.component.html

```diff
@@ -1,23 +1,44 @@
 <mat-toolbar>
-  <mat-toolbar-row>
-    <button mat-icon-button (click)="goBack()" *ngIf="!isDialog"><mat-icon>arrow_back</mat-icon></button>
-    <span i18n>{myView, select, myPersonals {myPersonals} myLibrary {myLibrary} other {Library}}</span>
-    <mat-form-field class="font-size-1 mat-form-field-type-no-underline mat-form-field-dynamic-width collections-search margin-lr" *ngIf="myView !== 'myPersonals'">
-      <planet-tag-input [formControl]="tagFilter" [db]="dbName" [parent]="parent" [filteredData]="resources.filteredData" [helperText]="false" [largeFont]="true" mode="filter" [selectMany]="false"></planet-tag-input>
-    </mat-form-field>
-    <span class="toolbar-fill margin-lr-10"></span>
-    <span class="font-size-1 margin-lr-4" i18n>Search by:</span>
-    <button mat-stroked-button i18n (click)="toggleFilters()" *ngIf="myView !== 'myPersonals'">Filter</button>
-    <mat-form-field class="font-size-1 margin-lr-4">
-      <input matInput i18n-placeholder placeholder="Title" [(ngModel)]="titleSearch">
-    </mat-form-field>
-    <button mat-raised-button color="primary" (click)="resetFilter()" [disabled]="resources.filter.trim() === '' && tagFilter.value.length === 0 && searchSelection._empty"><span i18n>Clear</span></button>
-  </mat-toolbar-row>
+  <ng-container *ngIf="deviceType !== deviceTypes.MOBILE; else mobileView">
+    <mat-toolbar-row>
+      <ng-container *ngTemplateOutlet="collectionRow"></ng-container>
+      <span class="toolbar-fill margin-lr-10"></span>
+      <ng-container *ngTemplateOutlet="filterRow"></ng-container>
+    </mat-toolbar-row>
+  </ng-container>
+
+  <ng-template #mobileView>
+    <mat-toolbar-row>
+      <ng-container *ngTemplateOutlet="collectionRow"></ng-container>
+    </mat-toolbar-row>
+    <mat-toolbar-row>
+      <br/>
```

Comment:
Is there any reason for putting BR tag here?

Reply:
Removed